local/debian/Makefile

```diff
@@ -0,0 +1,27 @@
+IMG ?= saucelabs/forwarder:debian
+CONTAINER ?= forwarder-debian
+
+.PHONY: build
+build:
+	@podman build -t $(IMG) .
+
+.PHONY: package
+package: DIST ?= ../../dist/forwarder*linux_arm64.deb
+package:
+	@cd ../../ && ./bin/goreleaser release --snapshot --skip-docker --clean
+	@cp $(DIST) forwarder.deb
+
+.PHONY: run
+run:
+	@podman run -p 3128:3128 -d -v ./forwarder.deb:/forwarder.deb --name $(CONTAINER) $(IMG)
+	@podman exec $(CONTAINER) dpkg -i /forwarder.deb
+	@podman exec $(CONTAINER) systemctl start forwarder
+	@podman exec -ti $(CONTAINER) /bin/sh
```

Comment:
Can we have it like run -it so that there is no need for stop?

Reply:
The entrypoints for systemd to work must be `/sbin/init`, or `/lib/systemd/systemd`.
```
If the command you running inside of the container is systemd or init, podman will setup tmpfs mount points in the following directories...
```

Running init requires user to log in and some additional setup, thus I made this little workaround.

Reply:
How about we remove the container once the `podman exec -ti $(CONTAINER) /bin/sh` exit?